proof/crefine/RISCV64/Finalise_C.thy

```diff
@@ -1732,22 +1732,6 @@ lemma cnotification_relation_udpate_arch:
   apply (safe ; case_tac "xa = p" ; clarsimp simp: option_map2_def map_option_case)
   done
 
-lemma sanitiseSetRegister_ccorres:
```

Comment:
I removed this lemma on RISCV64 and X64 because I noticed that it was unused. I think it was originally a helper lemma in ARM_HYP (where it is still used) and was copied over accidentally.
If it's too unrelated to the rest of the PR I'm happy to either revert or to leave it as a separate commit.

Reply:
separate commit at least. might also want to check other arches to see if we blundered there too